api-specs/Gateway-EE/3.3/kong-ee-3.3.yaml

```diff
@@ -8057,9 +11401,21 @@ tags:
     name: Key-sets
   - description: Workspaces
     name: Workspaces
-  - description: All about Basic-auth credentials in Kong
-    name: Basic-auth credentials
   - description: License
     name: licenses
   - description: Consumer Groups
-    name: consumer_groups
\ No newline at end of file
+    name: consumer_groups
+  - description: Information
+    name: Information
+  - description: Admins
+    name: Admins
+  - description: Groups
+    name: Groups
+  - description: Debug
+    name: Debug
+  - description: RBAC
+    name: RBAC
+  - description: Audit logs
+    name: Audit-logs
+  - description: Event hooks
+    name: Event-hooks
```

Comment:
Could we keep the format of tag names consistent?

Reply:
What do you mean?
I didn't get it.

Reply:
These names `Event-hooks`, `Audit-log` are consistent with the latest version.